docs/messageData.md

```diff
@@ -466,9 +466,9 @@ Last but not least, the greeting message contains a list of peers that tells cli
             "$comment": "Note: the string is encoded in Base64"
         },
         "address": {
-            "description": "Canonical address of the server with a protocol prefix and the port number",
+            "description": "Canonical address of the server with a protocol prefix and (optionally) the port number",
             "type": "string",
-            "pattern": "^.*:\\/\\/.*:\\d{0,5}\\/.*$"
+            "pattern": "^(ws|wss):(\\/\\/)[a-zA-Z0-9|.|-|_)+\\.[a-zA-Z]+((:[0-9]{1,5}(\\/.*)?)|(\\/).*|)$"
```

Comment:
I don't know if you copy pasted the regex, but if you did and it doesn't work, it is expect. In a json string we have to double escape so that `\/`  is treated as whole in the string instead of being interpreted as an escaped `/` , or something along these lines. Hence the double escape have to be converted to single escape on website and such. 
The CI currently fails for another reason. The regex I copied from the website I was testing it on came with some funny invisible characters....

Reply:
Yeah I didn't consider the double escape problem of the json string, so it looks good then 👍